airbyte-integrations/connectors/source-mailgun/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,129 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "domains",
+        "json_schema": {
+          "type": "object",
```

Comment:
json_schema can be empty in configured_catalog, it will be picked up from discover command by SAT